<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_TM</class>
 <widget class="QWidget" name="Create_TM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="Create_TM_text" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="Terminology_name">
        <property name="text">
         <string>术语库名称</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_Terminology_name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Terminology_descripte">
        <property name="text">
         <string>描述</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="textEdit_Terminology_descripte"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="Create_TM_tool" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Create_Terminology_sure">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Create_Terminology_Cancle">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
